docs/notebooks/demo/gretel-sagemaker-jumpstart-codellama.ipynb

```diff
@@ -0,0 +1,593 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "019c4fcd-d6c5-4381-8425-1d224c0ac197",
+   "metadata": {
+    "collapsed": true,
+    "jupyter": {
+     "outputs_hidden": true
+    },
+    "tags": []
+   },
+   "source": [
+    "## Setup\n",
+    "\n",
+    "### Install Necessary Packages\n",
+    "Please restart the kernel after executing the cell below for the first time."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e0e9f032-3eeb-4d4d-b4b2-1b150fa64cd6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%pip install --upgrade --quiet datasets transformers func_timeout"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b61aeb0a-0bfd-42d9-9277-fdfe57441752",
+   "metadata": {},
+   "source": [
+    "### Import Libraries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b1eea7db-d79d-4390-a51a-28442478953e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "import os\n",
+    "import pandas as pd\n",
+    "from datasets import load_dataset\n",
+    "from IPython.display import display\n",
+    "from IPython.display import HTML\n",
+    "from sagemaker import Session\n",
+    "from sagemaker.jumpstart.estimator import JumpStartEstimator\n",
+    "from sagemaker.s3 import S3Uploader\n",
+    "from transformers import AutoTokenizer\n",
+    "from tqdm.notebook import tqdm"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a3d21e27-a147-4268-b326-4a758e31c148",
+   "metadata": {},
+   "source": [
+    "### Select Model\n",
+    "Select your desired model ID. You can search for available models in the [Built-in Algorithms with pre-trained Model Table](https://sagemaker.readthedocs.io/en/stable/doc_utils/pretrainedmodels.html)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c7e01401-82db-4d49-9457-f930f4138618",
+   "metadata": {
+    "jumpStartAlterations": [
+     "modelIdOnly"
+    ],
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "model_id = \"meta-textgeneration-llama-codellama-13b\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5e19e16f-d459-40c6-9d6b-0272938b3878",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Dataset preparation\n",
+    "\n",
+    "### Load Dataset\n",
+    "We will use the [synthetic text-to-SQL dataset] (https://huggingface.co/datasets/gretelai/synthetic_text_to_sql) provided by Gretel. The dataset is a rich dataset of high-quality synthetic Text-to-SQL samples, designed and generated using Gretel Navigator, and released under Apache 2.0. \n",
```

Comment:
I think the space is messing with the link formatting. 

[synthetic text-to-SQL dataset] (https://huggingface.co/datasets/gretelai/synthetic_text_to_sql) -> [synthetic text-to-SQL dataset](https://huggingface.co/datasets/gretelai/synthetic_text_to_sql)

Reply:
Thanks!